res/controllers/DJ TechTools MIDI Fighter Twister.midi.xml

```diff
@@ -169,6 +169,26 @@
           <normal/>
         </options>
       </control>
+      <control>
+        <group>[Channel1]</group>
+        <key>volume</key>
```

Comment:
Is there no MIDI output for these? Or did you forget to add changes to the JS file to your commit?

Reply:
oh I forgot it https://github.com/mixxxdj/mixxx/pull/1778/commits/c219a6a1812a76e01b690e134610b948f5bb9da7